lib/avro_turf/confluent_schema_registry.rb

```diff
@@ -117,6 +133,7 @@ def delete_subject(subject)
   private
 
   def get(path, **options)
+    options = retry_options.merge!(options)
```

Comment:
```suggestion
    options.merge!(retry_options)
```

Reads a bit weird otherwise, because the bang! method mutates the object on which it's called